consensus/src/simplex/actors/batcher/round.rs

```diff
@@ -57,15 +56,8 @@ impl<
         R: Reporter<Activity = Activity<S, D>>,
     > Round<P, S, B, D, R>
 {
-    pub fn new(participants: Ordered<P>, scheme: S, blocker: B, reporter: R, batch: bool) -> Self {
-        // Configure quorum params
-        let quorum = if batch {
```

Comment:
Now that we know participants during init, we don't need to do this `initialized` check.